docs/conf.py

```diff
@@ -150,15 +150,15 @@ def _compute_navigation_tree(context):
 # Find the current builder
 builder = "dirhtml"
 if '-b' in sys.argv:
-    builder = sys.argv[sys.argv.index('-b')+1]
+    builder = sys.argv[sys.argv.index('-b') + 1]
 
 html_theme = 'furo'
 html_last_updated_fmt = ""
 html_permalinks_icon = "¶"
 html_theme_options = {
     "light_css_variables": {
         "color-sidebar-background-border": "none",
-        "font-stack": "Ubuntu, -apple-system, Segoe UI, Roboto, Oxygen, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif",
+        "font-stack": "Ubuntu, -apple-system, Segoe UI, Roboto, Oxygen, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif",  # noqa: E501
```

Comment:
This is just [line is too long](https://docs.astral.sh/ruff/rules/line-too-long/) and could be wrapped easily enough, but it seemed like this comes from somewhere rather than is hand-written so I thought leaving it untouched was perhaps best.